### DISCUSSION TITLE
"Dati riconoscimento" su matr_009

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1006] [Supporto servizi cooperativi]
Nel caso d'uso in oggetto, a quali atto di riconoscimento dovrebbero fare riferimento le informazioni:

![image](https://github.com/user-attachments/assets/a5dc5fa2-1837-42ad-8292-117a197673c8)

?

**Comment by acolosimosogei** [comment]
Salve, con "Dati Riconoscimento" su matr_009 si fa riferimento all'atto di riconoscimento del figlio il quale si presume sia stato emesso precedentemente al matrimonio.

Chiudiamo pertanto la discussion: in caso di ulteriori richieste si prega di aprirne un'altra dedicata.
Grazie.

**Comment by acolosimosogei** [comment]
Salve, 

confermo che la sezione "Dati riconoscimento" per il caso d'uso discusso fa riferimento al caso da lei indicato come caso b.

Grazie.

**Comment by antonioluberto** [comment]
Mi scuso, ma ancora qualcosa non è chiaro:
per la sezione 

![image](https://github.com/user-attachments/assets/91b3db47-d388-44fc-aa58-671cd680ee63)

si indica in effetti come controllo di obbligatorietà:

![image](https://github.com/user-attachments/assets/65f550e6-7dab-4f72-8491-0cb7439f4cd7)

dal che discende che la sezione:
- può essere non popolata se il figlio non è pre-morto
- deve necessariamente essere popolata se il figlio è pre-morto

Il riferimento pertanto è ad un ipotetico _atto di riconoscimento_ legato alla condizione per la quale il "figlio"  sia (pre-)morto al momento della redazione dell'atto di matrimonio.

E allora, a quale atto di riconoscimento (di "chi" nei confronti di "chi") si sta effettivamente facendo riferimento?

Rimaniamo in attesa di riscontro

**Comment by acolosimosogei** [comment]
Ciao, 

con la formula di obbligatorietà: "evento.datiEventoMatrimonio.datiRiconoscimento[0].figlioPremorto,=,false", si deve intendere come obbligatorio che il figlio NON sia premorto per la sezione "Dati Riconoscimento", in quanto la sezione fa riferimento al figlio.

Grazie

**Comment by antonioluberto** [comment]
Buonasera.
Avendo rilevato un errore nella composizione del mio ultimo quesito, riformulo la domanda qui di seguito:

la documentazione indica:

![image](https://github.com/user-attachments/assets/fdc2898c-b0c6-4efb-8b63-f368c2214a45)

dal che discende esplicitamente che la sezione "Dati riconoscimento" deve essere popolata (obbligatoriamente) se
**evento.datiEventoMatrimonio.datiRiconoscimento[0].figlioPremorto = true**.

Gentilmente: quale atto di riconoscimento  (di "chi" nei confronti di "chi")  dovremmo inserire nella sezione "Dati riconoscimento" laddove  il "figlio" sia (pre-)morto al momento della redazione dell'atto di matrimonio?

**Comment by acolosimosogei** [comment]
Salve,

per la sezione discussa: "Dati riconoscimento", c'è una condizione che la rende **sempre** opzionale, procederemo a modificare la documentazione per renderla più chiara in tal senso.

Grazie per la segnalazione

**Comment by acolosimosogei** [comment]
Salve, 

è stata aggiornata la documentazione, provvedo a chiudere la discussion.

Grazie!
